<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="mgo_icon">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="materialiser_btn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Open Materialiser (import / export material presets)</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Export</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Output Folder</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="outputPath_edit">
                  <property name="toolTip">
                   <string>Output path for exporting mGo data</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="browseDir_btn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Set output folder (for export of textures and mGo description file)</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QCheckBox" name="exportChannels_chk">
                  <property name="toolTip">
                   <string>Export Texture CHANNELS</string>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="exportAttrs_chk">
                  <property name="toolTip">
                   <string>Export shader ATTRIBUTES</string>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="exportGeo_chk">
                  <property name="toolTip">
                   <string>Export object GEOMETRY</string>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::MinimumExpanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="sendMode_cbox">
                  <property name="toolTip">
                   <string>Export Method:
Selected Object - Exports checked data associated with currently selected object
Visible Objects - Exports checked data associated with all visible objects
All Objects - Exports checked data associated with all objects
Env &amp; Cam - Exports Perspective Viewport Camera and Environment Light as an HDRI dome/sky light</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Current Object</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Visible Objects</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>All Objects</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_38">
                <item>
                 <spacer name="horizontalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="exportDescription_btn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Export as mGo description to output folder</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sendToApp_btn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Go! (Export checked items to 3D package)</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>289</width>
            <height>516</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Host</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Host</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="appHost_cbox">
                    <item>
                     <property name="text">
                      <string>Local Host Only</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>25</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>25</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Export Application</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QComboBox" name="exportApp_cbox">
                    <item>
                     <property name="text">
                      <string>Maya</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>3ds Max</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Softimage</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Texture Export Options</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_21">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>8-Bits</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="eightBitFormat_cbox">
                    <item>
                     <property name="text">
                      <string>tif</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>bmp</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>jpg</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>jpeg</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>png</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ppm</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>psd</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>tga</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>tiff</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>xbm</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>xpm</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>16/32 Bits</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="thirtyTwoBitFormat_cbox">
                    <item>
                     <property name="text">
                      <string>exr</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>psd</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>tif</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>tiff</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>Texture Import Options</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_42">
                  <item>
                   <widget class="QCheckBox" name="checkBox">
                    <property name="text">
                     <string>Create Gamma Correction Nodes</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_41">
                  <item>
                   <widget class="QLabel" name="label_24">
                    <property name="text">
                     <string>Filter</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="filterType_cbox">
                    <item>
                     <property name="text">
                      <string>None</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Mipmap</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Quadratic</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Animation</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Mipmap Conversion</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Preset</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_3"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_9">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>25</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>25</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>+</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_16">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Tool Path</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_5"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="browse_output_btn_5">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>M</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_23">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>Options</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_6"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_24">
                    <item>
                     <widget class="QLabel" name="label_19">
                      <property name="text">
                       <string>Subfolder</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_7"/>
                    </item>
                    <item>
                     <widget class="QComboBox" name="comboBox_20">
                      <item>
                       <property name="text">
                        <string>tx</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>exr</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="maximumSize">
                       <size>
                        <width>50</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Save</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
